docs/competition/matches/match-scoring.md

```diff
@@ -18,36 +18,51 @@ Note that this role is _not_ about making decisions related to the interpretatio
 
 **Responsible for being in the arena, saving robots and recording the end-state of matches. They may not bring in any subordinate volunteers.**
 
-In this role you will be assigned an arena and work with a partner. You will be allowed to enter the arena during matches if it is deemed necessary, however it is very important that you do not obstruct any robot or its vision and leave the arena as soon as possible. Each of you will be each given a score sheet<sup>1</sup> on a clipboard which you must both individually fill in fully by the end of the match. Anything that happens after the match has ended must not be counted as a score and the arena must not be changed until both scorers have finished. Once a match has finished it is your responsibility to take your completed score sheet to the score entry desk which will be near the arenas and you must return promptly to the arena in case you need to be found for further scoring questions. Teams are not allowed to enter the arena to collect their robots until after scoring has finished.
+You MUST read [the rules](https://www.studentrobotics.org/docs/rules/) before starting this role, as you will be enforcing them. Especially make sure you understandthe scoring rules (i.e. what counts as 'in' a zone), and how the arena should be laid out.
```

Comment:
We should provide training and ideally a paper copy of the relevant regulations for these volunteers.

We should _not_ rely on them having read the rulebook.

Reply:
For clarity: I think it's reasonable to ask that they have read the rules though.

Reply:
I'll update the doc with a summary of the most required rules.

Reply:
Fixed in 0a0f4290631d8624f829ef9561bf1597cb580ae9